app/src/main/java/fr/free/nrw/commons/contributions/Contribution.java

```diff
@@ -68,6 +68,7 @@ public Contribution createFromParcel(Parcel parcel) {
     public String decimalCoords;
     public boolean isMultiple;
     public String wikiDataEntityId;
+    public String p18Value;
```

Comment:
Why is it public when it has a getter and a setter?

Reply:
Very true, sorry ı just ctrl+D from previous line, will fix it ASAP:)